### DISCUSSION TITLE
根据requirements.txt文件安装的环境有问题

### DISCUSSION THREAD

**Discussion by ajsterling123** [zai-org/ChatGLM3#1322] [Bad Case]
transformer库一直不兼容

**Comment by IvyWang0629** [comment]
同遇到requirements.txt问题，vllm到底应该安装哪个版本，系统一直从vllm-0.5.1.tar.gz下载并编译，与其他包冲突后一直向前版本回退（0.5.1->0.5.0->0.4.3->0.4.2）。可耗时间了
